code/modules/research/xenobiology/crossbreeding/crystalized.dm

```diff
@@ -1,106 +1,138 @@
-		return
-	new crystal_type(user_turf)
-	qdel(src)
-
-/obj/item/slimecross/crystalized/grey
-	crystal_type = /obj/structure/slime_crystal/grey
-	colour = "grey"
-
-/obj/item/slimecross/crystalized/orange
-	crystal_type = /obj/structure/slime_crystal/orange
-	colour = "orange"
-
-/obj/item/slimecross/crystalized/purple
-	crystal_type = /obj/structure/slime_crystal/purple
-	colour = "purple"
-
-/obj/item/slimecross/crystalized/blue
-	crystal_type = /obj/structure/slime_crystal/blue
-	colour = "blue"
-
-/obj/item/slimecross/crystalized/metal
-	crystal_type = /obj/structure/slime_crystal/metal
-	colour = "metal"
-
-/obj/item/slimecross/crystalized/yellow
-	crystal_type = /obj/structure/slime_crystal/yellow
-	colour = "yellow"
-
-/obj/item/slimecross/crystalized/darkpurple
-	crystal_type = /obj/structure/slime_crystal/darkpurple
-	colour = "dark purple"
-
-/obj/item/slimecross/crystalized/darkblue
-	crystal_type = /obj/structure/slime_crystal/darkblue
-	colour = "dark blue"
-
-/obj/item/slimecross/crystalized/silver
-	crystal_type = /obj/structure/slime_crystal/silver
-	colour = "silver"
-
-/obj/item/slimecross/crystalized/bluespace
-	crystal_type = /obj/structure/slime_crystal/bluespace
-	colour = "bluespace"
-
-/obj/item/slimecross/crystalized/sepia
-	crystal_type = /obj/structure/slime_crystal/sepia
-	colour = "sepia"
-
-/obj/item/slimecross/crystalized/cerulean
-	crystal_type = /obj/structure/slime_crystal/cerulean
-	colour = "cerulean"
-
-/obj/item/slimecross/crystalized/pyrite
-	crystal_type = /obj/structure/slime_crystal/pyrite
-	colour = "pyrite"
-
-/obj/item/slimecross/crystalized/red
-	crystal_type = /obj/structure/slime_crystal/red
-	colour = "red"
-
-/obj/item/slimecross/crystalized/green
-	crystal_type = /obj/structure/slime_crystal/green
-	colour = "green"
-
-/obj/item/slimecross/crystalized/pink
-	crystal_type = /obj/structure/slime_crystal/pink
-	colour = "pink"
-
-/obj/item/slimecross/crystalized/gold
-	crystal_type = /obj/structure/slime_crystal/gold
-	colour = "gold"
-
-/obj/item/slimecross/crystalized/oil
-	crystal_type = /obj/structure/slime_crystal/oil
-	colour = "oil"
-
-/obj/item/slimecross/crystalized/black
-	crystal_type = /obj/structure/slime_crystal/black
-	colour = "black"
-
-/obj/item/slimecross/crystalized/lightpink
-	crystal_type = /obj/structure/slime_crystal/lightpink
-	colour = "light pink"
-
-/obj/item/slimecross/crystalized/adamantine
-	crystal_type = /obj/structure/slime_crystal/adamantine
-	colour = "adamantine"
-
-/obj/item/slimecross/crystalized/rainbow
-	crystal_type = /obj/structure/slime_crystal/rainbow
-	colour = "rainbow"
+/*
+Crystalized extracts:
+	Creates a crystal structure that
+	grants unique properties in an aura around it
+	when activated in-hand.
+*/
+
+/obj/item/slimecross/crystalized
+	name = "crystalized extract"
+	desc = "It's crystalline,"
+	effect = "crystalline"
+	icon_state = "crystalline"
+	var/obj/structure/slime_crystal/crystal_type
+
+/obj/item/slimecross/crystalized/Initialize()

```

Comment:
unneeded, everything does this by default.
If this doesn't it's been overwritten somewhere else and this would just cause confusion.